test/functional/gridfs_stream_tests.js

```diff
@@ -1190,7 +1190,15 @@ describe('GridFS Stream', function() {
                 download.on('end', function() {
                   var result = testSpec.assert.result;
                   if (!result) {
-                    test.ok(false);
+                    // test.ok(false);
```

Comment:
Can we change this to the `expect(result).to.not.exist` format we talked about earlier?

Reply:
done